scripts/check-deps/check-dependencies.js

```diff
@@ -0,0 +1,324 @@
+/* eslint no-for-of-loops/no-for-of-loops: 0 */
```

Comment:
For all of these files that are straight copies, can we add something in the comment block above that says something like

> File copied directly from https://github.com/wincent/js/blob/master/packages/workspace-scripts/src/dependencies.ts

This way we know exactly where its from and then also not any deviation from that copy.

Reply:
FWIW, I don't think you really need it anywhere in the file, just mentioning it in the commit message is enough. (Consider it a fork, but one where it doesn't matter if it ever deviates and it won't have to track the upstream.)

Normally when you copy code you'd have to worry about licensing and preserving headers and all that (this project is BSD licensed; that repo is MIT licensed), but in this case I'm the original author and I told @diegonvs to do whatever he wants with the code 😂.

*Signed, @wincent* 🖊 

About license headers though, I think this ESLint suppression should go below the SPDX license header; we always want that to be first.

Reply:
I'd be fine with just a commit message as well. Primarily I just want us to be able to know "oh this is forked from this existing file".

Reply:
@bryceosterhaus @wincent see 4e00e86

Reply:
It's enough?

Reply:
Hmm, could we maybe point to the exact file instead of just the repo? just since they are named differently

Reply:
Seeing as we're having a great time talking about this, can we also switch `0` to `'off'` (because it is a bit more explicit), or better yet:

```javascript
/* eslint-disable no-for-of-loops/no-for-of-loops */
```

For consistency with what we mostly seem to do:

- [Hits for "eslint-disable" in liferay-portal](https://github.com/liferay/liferay-portal/search?q=eslint-disable&unscoped_q=eslint-disable)
- [Hits for "eslint-disable" in this repo](https://github.com/liferay/clay/search?q=eslint-disable&unscoped_q=eslint-disable) (albeit with some inconsistency there in `IncrementalInteractions.tsx`).